### PR TITLE
Terraform 0.14 upgrade guide

### PR DESCRIPTION
**Pull request by brikis98**
This is the upgrade guide for Terraform 0.14. Note that we are [still working on 4 repos](https://www.notion.so/gruntwork/Terraform-0-14-Upgrade-87f06063b1bd46789a6e0089168674d6), so their version numbers are currently listed as TBD.

### CODE REVIEW

**Comment by infraredgirl** [comment]
Looks good - will approve as soon as all the version numbers are filled in!

**Comment by brikis98** [comment]
OK, all version numbers are filled in! This is ready for final review.

**Review by infraredgirl — APPROVED** [review]
LGTM!

**Comment by brikis98** [comment]
Thanks for the quick reviews! Merging now.

**Comment by brikis98** [comment]
https://github.com/gruntwork-io/gruntwork-io.github.io/releases/tag/v0.0.70

File: _posts/2021-04-13-how-to-update-to-terraform-14.adoc [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+[[deployment_walkthrough]]
+== Deployment walkthrough
+
+=== Step 1: update your code to be compatible with Terraform 0.13
+
+If you haven't already, you need to:
+
+. Update your code to work with Terraform 0.13. Do NOT skip from, say, 0.11, straight to 0.14. You MUST update to
+  0.13.0 or above first! If you're still on Terraform 0.11 or older, see our
```
Comment (marinalimeira) [inline _posts/2021-04-13-how-to-update-to-terraform-14.adoc:77]: This notation "0.13.0" is confusing for me. Perhaps "0.13.x" is better?

Comment (brikis98) [reply _posts/2021-04-13-how-to-update-to-terraform-14.adoc:77]: Why is 0.13.0 confusing? That's the first release on the 0.13.x line and is the minimum version allowable here...

**Pull request merged**